.config-schema.json

```diff
@@ -120,6 +120,17 @@
             "default": [],
             "x-env-variable": "OPENFGA_EXPERIMENTALS"
         },
+        "checkTrackerEnabled": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Enable/disable tracking tuples.",
```

Comment:
Could you improve this description so that users can know whether they want it enabled or disabled? 

What does "tracking tuples" mean? Hhat information does it track, how is it exposed for consumption (logs), what can one do with it afterwards?

Reply:
👍